app/mixins/tooltip-for-dropdown.js

```diff
@@ -0,0 +1,25 @@
+import Mixin from '@ember/object/mixin';
+import { get, observer, set } from '@ember/object';
+
+export default Mixin.create({
+  dropdownOpen: false,
+  tooltipShown: false,
+
+  mouseEnter() {
+    let dropdownOpen = get(this, 'dropdownOpen');
+    if (dropdownOpen) {
+      return;
+    } else {
+      set(this, 'tooltipShown', true);
+    }
+  },
+
+  mouseLeave() {
+    set(this, 'tooltipShown', false);
+  },
+
+  dropdownOpened: observer('dropdownOpen', function() {
+    let tooltipShown = get(this, 'dropdownOpen') ? false : null;
```

Comment:
On this line alone, I think this mixin needs some documentation. 

Probably a description of how the tooltip ought to behave in relation to the dropdown.